corehq/apps/data_cleaning/views/tables.py

```diff
@@ -147,12 +147,24 @@ def apply_all_changes(self, request, *args, **kwargs):
     @hq_hx_action("post")
     def undo_last_change(self, request, *args, **kwargs):
         self.session.undo_last_change()
-        return self.get(request, *args, **kwargs)
+        return self._trigger_clean_form_referesh(
+            self.get(request, *args, **kwargs)
+        )
 
     @hq_hx_action("post")
     def clear_all_changes(self, request, *args, **kwargs):
         self.session.clear_all_changes()
-        return self.get(request, *args, **kwargs)
+        return self._trigger_clean_form_referesh(
+            self.get(request, *args, **kwargs)
+        )
+
+    def _trigger_clean_form_referesh(self, response):
```

Comment:
nit: spelling (and the places it's used above)
```suggestion
    def _trigger_clean_form_refresh(self, response):
```